addon/utils/bind-data-test-attributes.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+const TEST_SELECTOR_PREFIX = /data-test-.*/;
+
+export default function bindDataTestAttributes(component) {
+  let computedBindings = component.attributeBindings && component.attributeBindings.isDescriptor;
+  if (computedBindings) {
+    let message = `ember-test-selectors could not bind data-test-* properties on ${component} ` +
+      `automatically because attributeBindings is a computed property.`;
+
+    return Ember.warn(message, false, {
+      id: 'ember-test-selectors.computed-attribute-bindings',
+    });
```

Comment:
If we could somehow make it work correctly in this case as well that would be awesome of course. In theory it should be possible to replace the CP with a wrapper one that we create of course but I'm not sure it's worth the added complexity.